Andrea / exercise-1/index.html

```diff
@@ -0,0 +1,181 @@
+                    </p>
+                    
+                    <h3>finding books you like</h3>
+                    <p>
+                      Lorem ipsum dolor, sit amet consectetur adipisicing elit. 
+                      Cupiditate, voluptatum ut assumenda doloremque, velit dolor
+                      ab nemo placeat aspernatur quam, voluptatibus nulla explicabo ver
+                    </p>
+                </div>
+            <!-- </section> -->
+          
+        <aside class="recommendations">
+            <h4>recommendations</h4>
+            <ul>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>the name of the wind <span>5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>shadowhunters <span>3.5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png"" alt="icon">
+                    <p>the song of achilles <span>5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>a little life <span> 5/5</span></p>
+                </li>
+                <li>
+                    <img src="icon.png" alt="icon">
+                    <p>pride and prejudice<span> 5/5</psan></p>
+                </li>
+            </ul>
+        </aside>
+    </div>
+    </main>
+
+    <footer>
+        <nav class="footer-menu">
+            <img class="logo" src="logo.png" alt="image">
+            <ul class="footer-menu__links">
+                <li>
+                    <a href="#">travel</a>
+                </li>
+                <li>
+                    <a href="#">explore</a>
+                </li>
+                <li>
+                    <a href="#">works</a>
+                </li>
+                <li>
+                    <a href="#">about us</a>
+                </li>
+                <li>
+                    <a href="#">contacts</a>
+                </li>
+                <!-- Botones -->
+                <li>
+                    <input class="footer-menu__input" type="text" placeholder="Your email">
+                </li>
+                <li>
+                    <a class="footer-menu__button" href="#">send</a>
+                </li>
+            </ul>
+            
+        </nav>
+            
+        <nav class="footer-menu">
+            <ul class="footer-menu__links">
+                    <li>
+                        <a href="#">terms</a>
+                    </li>
+                    <li>
+                        <a href="#">promo</a>
+                    </li>
+                    <li>
+                        <a href="#">download</a>
+                    </li>
+                    <li>
+                        <a href="#">news</a>
+                    </li>     
+              
+                <!-- Iconos Redes Sociales--> 
+                <div class="footer-menu__social">
```

Comment:
lets make this a new UL sibling of the current UL it is contained

Reply:
make this div a sibling ul instead. You will end up with 2 ULs as children of the `<nav>` and is ok